### PR TITLE
[ISSUE #3046] Unify the treatment of constant variables and reduce the usage of "magic numbers"

### PR DESCRIPTION
**Pull request by ArthurMograbi**
## What is the purpose of the change
To resolve issue #3046, unify the treatment of constant variables and reduce the usage of "magic numbers"

## Brief changelog
Replacement of some of the usage of "magic numbers" for a cleaner and more maintainable approach and unify some discordand usages of the same variables

## Verifying this change
eclipse-pmd was used to verify the changes

### CODE REVIEW

File: client/src/main/java/com/alibaba/nacos/client/config/impl/ClientWorker.java [hunk 1]
```diff
@@ -113,7 +113,8 @@
     /**
      * groupKey -> cacheData.
      */
-    private final AtomicReference<Map<String, CacheData>> cacheMap = new AtomicReference<>(new HashMap<>());
+    private final AtomicReference<Map<String, CacheData>> cacheMap = new AtomicReference<Map<String, CacheData>>(
```
Comment (KomachiSion) [inline client/src/main/java/com/alibaba/nacos/client/config/impl/ClientWorker.java:116]: Why add type?

**Review by KomachiSion — CHANGES_REQUESTED** [review]
I think your jdk version is too old. Please upgrade jdk version to at lease 1.8.

**Comment by ArthurMograbi** [comment]
> I think your jdk version is too old. Please upgrade jdk version to at lease 1.8.

Thank you for the heads up! There was something wrong with my workspace that caused java to default to an earlier version

**Comment by realJackSun** [comment]
1 Do not use magic number
2 Remove the types as sion says

**Comment by KomachiSion** [comment]
CLA not sign for a long time.

**Pull request closed without merging**